tests/integration_test/run_int_test_locally.sh

```diff
@@ -3,39 +3,56 @@
 SECONDARY_ANALYSIS_REPO_ROOT=$1
 VAULT_TOKEN_PATH=${VAULT_TOKEN_PATH:-"${HOME}/.vault-token"}
 
-#ENV=$1
-#LIRA_MODE=$2
-#LIRA_VERSION=$3
-#PIPELINE_TOOLS_MODE=$4
-#PIPELINE_TOOLS_VERSION=$5
-#TENX_MODE=$6
-#TENX_VERSION=$7
-#SS2_MODE=$8
-#SS2_VERSION=$9
-#TENX_SUB_ID=${10}
-#SS2_SUB_ID=${11}
-#VAULT_TOKEN_PATH=${12}
-#SUBMIT_WDL_DIR=${13}
-#USE_CAAS=${14}
-#USE_HMAC=${15}
-#SUBMIT_AND_HOLD=${16}
+#ENV=${1}
+#LIRA_MODE=${2}
+#LIRA_VERSION=${3}
+#LIRA_DIR=${4}
+#SECONDARY_ANALYSIS_MODE=${5}
+#SECONDARY_ANALYSIS_VERSION=${6}
+#SECONDARY_ANALYSIS_DIR=${7}
+#PIPELINE_TOOLS_MODE=${8}
+#PIPELINE_TOOLS_VERSION=${9}
+#PIPELINE_TOOLS_DIR=${10}
+#TENX_MODE=${11}
+#TENX_VERSION=${12}
+#TENX_DIR=${13}
+#SS2_MODE=${14}
+#SS2_VERSION=${15}
+#SS2_DIR=${16}
+#TENX_SUB_ID=${17}
+#SS2_SUB_ID=${18}
+#VAULT_TOKEN_PATH=${19}
+#SUBMIT_WDL_DIR=${20}
+#USE_CAAS=${21}
+#USE_HMAC=${22}
+#SUBMIT_AND_HOLD=${23}
+#COLLECTION_NAME=${24:-"lira-${LIRA_ENVIRONMENT}"}
```

Comment:
Same as above, there's no `LIRA_ENVIRONMENT` variable defined in this script

Reply:
Changed it to just ENVIRONMENT, which is a variable defined in the gitlab.yml file